docs/src/pages/system/visually-hidden/VisuallyHiddenUsage.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { visuallyHidden } from '@material-ui/system';
+import { makeStyles } from '@material-ui/core/styles';
+import ThumbUpIcon from '@material-ui/icons/ThumbUp';
+import EmojiEmotionsIcon from '@material-ui/icons/EmojiEmotions';
+import FavoriteIcon from '@material-ui/icons/Favorite';
+
+const useStyles = makeStyles((theme) => ({
+  root: { 
+    border: 'none',
+    color: 'grey',
+    '& label': { marginRight: 10, cursor: 'pointer' }
+  },
+  selected: {
+    color: theme.palette.primary.main,
+  },
+  visuallyHidden,
+}));
+
+export default function VisuallyHiddenUsage() {
+  const classes = useStyles();
+  const [ selected, setSelected ] = React.useState('like');
+  
+  return (
+    <fieldset className={classes.root}>
```

Comment:
Added more realistic scenario where the `visuallyHidden` would be used

![image](https://user-images.githubusercontent.com/4512430/84577600-21c26180-adbe-11ea-97e4-e3977bcd079a.png)

cc @eps1lon let me know what do you think about this example, although it may be too complicated now.. :\

Reply:
I'm not sure about the demo, I can't use the keyboard to navigate the option, not notice any focusable style. I'm also not sure about the complexity of the demo. https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html seems to have a few nice use cases.

Reply:
Should we just add a simple example with an `<a />` element?, would be nice if it the code is short and immediately shown below the example. What do you think?

Reply:
I've added the link example, I can add another one for the date start and end example if you think this is not enough

![image](https://user-images.githubusercontent.com/4512430/84593377-406f3980-ae4c-11ea-8add-0c94ea383100.png)